GitOut/Features/Git/Log/GitLogPage.xaml

```diff
@@ -425,9 +442,25 @@
                         Placement="Bottom"
                     >
                         <Grid>
-                            <Border Style="{StaticResource CardPanelStyle}" MinWidth="180" Margin="16 8" Padding="0">
+                            <Grid.Resources>
+                                <Style x:Key="PopoverCheckboxListItemStyle" TargetType="{x:Type CheckBox}" BasedOn="{StaticResource MaterialCheckBoxStyle}">
+                                    <Setter  Property="Margin" Value="0 8" />
```

Comment:
adjust the two spaces to one

Reply:
Fixed the spaces and also updated the spacing for self-closing elements to be the same (without space)